featuregates/client/pkg/featuregateclient/client_test.go

```diff
@@ -71,6 +71,58 @@ func TestGetFeature(t *testing.T) {
 		})
 	}
 }
+func TestGetFeatureList(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), contextTimeout)
+	defer cancel()
+
+	objs, _, _ := fake.GetTestObjects()
+	s := scheme.Scheme
+	if err := corev1alpha2.AddToScheme(s); err != nil {
+		t.Fatalf("Unable to add config scheme: (%v)", err)
```

Comment:
nit: start error messages with lower case (there is inconsistency between files here).